src/Faq.tsx

```diff
@@ -180,47 +184,11 @@ const Faq: FunctionComponent<{ onJoin: () => void }> = ({ onJoin }) => {
                 <QuestionStyled>
-                    What studies are currently using Mobile Toolbox?
+                    What studies are currently using Open Bridge?
                 </QuestionStyled>
                 <AnswerStyled>
                     <p>
-                        The <a href="https://www.brainhealthregistry.org/" target="_blank"><strong>UC San Francisco Brain Health Registry</strong></a> study enrolls Brain
-                        Healthy Registry participants to establish national normative values
-                        for the Mobile Toolbox cognitive measures, and looks at feasibility,
-                        usability, and compliance for the Mobile Toolbox app in an
-                        unsupervised, remote setting in a diverse cohort of individuals.
-                        Launched February 2022
-                    </p>
-                    <p>
-                        <strong>Harvard Aging Brain Study</strong> deploys Mobile Toolbox into a
-                        well-characterized cohort of 100 cognitively normal (CN), older
-                        individuals who have in-clinic cognitive testing, including the
-                        Primary Alzheimer Cognitive Composite (PACC) and AD biomarkers on
-                        PET imaging. Participants will be recruited from 3 existing NIH
-                        funded studies (the Harvard Aging Brain Study, the IADL Study and
-                        the Subjective Cognitive Decline study).<strong> Launched April 2022.</strong>
-                    </p>
-                    <p>
-                        <strong>The Emory Healthy Brain Study (EHBS) and Emory Healthy Aging Study
-                            (EHAS)</strong> deploys Mobile Toolbox to participants divided into three
-                        groups based on cognitive functioning instrument (CFI) scores,
-                        matched for age, race, gender, and education in a series of steps.
-                        Launched April 2022.
-                    </p>
-                    <p>
-                        <strong>UC San Diego</strong> deploys Mobile Toolbox in a sample of persons who are
-                        living with HIV (PWH), with and without HIV-associated
-                        neurocognitive disorders (HAND). Launched April 2022.
-                    </p>
-                    <p>
-                        <strong>Washington University, St. Louis</strong> uses Mobile Toolbox in a cohort of
-                        brain-healthy individuals who originally participated in the Human
-                        Connectome Project. Launched March 2022.
-                    </p>
-                    <p>
-                        Sage Bionetworks uses Mobile Toolbox in 4 sub-populations of their
-                        existing mPower 2.0 digital health studies in Parkinson’s Disease
-                        populations.
+                        Lorem etc. etc.
```

Comment:
What do you think about adding a `TODO` comment here, so it's easily findable for update in the future?

Reply:
:+1: